packages/frontend/core/src/desktop/components/app-container/index.tsx

```diff
@@ -1,5 +1,6 @@
 import { useAppSettingHelper } from '@affine/core/components/hooks/affine/use-app-setting-helper';
 import { RootAppSidebar } from '@affine/core/components/root-app-sidebar';
+import { AppFallback } from '@affine/core/mobile/components';
```

Comment:
can we move this AppFallback to 
`packages/frontend/core/src/desktop/components/app-container/mobile.tsx`

And replace all uses of it with <AppContainer fallback/> ?

Reply:
Damn! but `<AppFallback />` has a mobile `WorkspaceSelector` inside!